app/src/main/java/com/example/fittracker/mvp/model/MainScreenModel.java

```diff
@@ -1,9 +1,39 @@
 package com.example.fittracker.mvp.model;
 
+import android.location.Location;
+
 import com.example.fittracker.mvp.contract.MainScreenContract;
+import com.example.fittracker.services.WeatherService;
 
 public class MainScreenModel implements MainScreenContract.Model {
+    private Location location;
+    private WeatherService service;
+
     public MainScreenModel() {
```

Comment:
here you must have the instance of the WeatherService.

Reply:
Done